src/consensus/pbft/pbft_tables.h

```diff
@@ -17,5 +17,8 @@ namespace pbft
 
   static constexpr auto replicate_type_pbft = kv::ReplicateType::SOME;
   static const std::unordered_set<std::string> replicated_tables_pbft = {
-    Tables::PBFT_REQUESTS, Tables::PBFT_PRE_PREPARES, Tables::PBFT_NEW_VIEWS};
+    Tables::PBFT_REQUESTS,
```

Comment:
Let's try to merge the requests table when we can, and rename the rest?

Reply:
This is part of  #1600